ring/src/ring_verifier.rs

```diff
@@ -12,28 +12,28 @@ use crate::piop::params::PiopParams;
 use crate::piop::{FixedColumnsCommitted, PiopVerifier, VerifierKey};
 use crate::RingProof;
 
-pub struct RingVerifier<F, CS, Curve, T>
+pub struct RingVerifier<F, CS, Jubjub, T>
```

Comment:
Why you decided to rename `Curve` to `Jubjub`?

Isn't "Jubjub" the specific name of the curve whose base field is the prime field of BLS12-381?
But you library is not limited to that specific use case, e.g. in ark-ec-vrfs I'm going to use this lib for all the possible curves defined by arkworks.

Just for my understanding, is Jubjub the "de-facto" standard name used for curves defined over the prime field of another pairing friendly curve? If not, maybe is worth to be more "abstract" about the generic name

Reply:
Yes, jubjub is a specific curve, but it's common (i think) to call a jubjub any TE curve on top a pairing-friendly curve (babyjubjub on bn254, bandersnatch, etc). So given it specializes to TE, i thought Jubjub is better because we have 2 curves.